gitium/gitium.php

```diff
@@ -322,9 +322,33 @@ function gitium_hook_plugin_and_theme_editor_page( $hook )
     return;
 }
 
-// Hook to theme/plugin edit page
+/*
+ * We execute the "gitium_auto_push" on "wp_die_ajax_handler" filter.
+ * This will get executed twice per request for some odd reason.
+ * Still, only one commit will be made since second time we don't have anything to commit.
+ */
+function gitium_check_ajax_success_call($callback)
+{
+    gitium_auto_push();
+}
```

Comment:
You need to return the `$callback` here. I guess that will solve the double calling.